src/site/content/en/blog/aria-poison-or-antidote/index.md

```diff
@@ -0,0 +1,283 @@
+`aria-expanded="false"` on each expandable menuitem to indicate that 1) there is something that can
+be expanded, and 2) it currently is not expanded. As an added touch the author should put
+`role="none"` on the img triangle to indicate it's for prettification purposes only. This prevents
+the screen reader from saying anything annoying about the image that would be redundant at best.
+
+## Dealing with bugs
+
+### Keyboard bugs (HTML!)
+
+Although keyboard access is a part of core HTML, authors mess it up all the time, either because
+they don't use keyboard navigation all that much, or because there is much nuance to get right. 
+
+Examples of bugs:
+
++   A checkbox uses spacebar to toggle, but the author forgot to call `preventDefault()`. Now
+    the spacebar will both toggle the checkbox and page down, which is the default browser behavior
+    for spacebar.
++   An ARIA modal dialog wants to trap tab navigation inside of it, and the author forgets to
+    specifically allow <kbd>Control</kbd>+<kbd>Tab</kbd> through to the browser. Now,
+    <kbd>Control</kbd>+<kbd>Tab</kbd> just navigates within their dialog, and doesn't switch tabs
+    in the browser as it should. Ugh.
++   An author creates a selection list, and implements up/down, but does not implement
+    home/end/pageup/pagedown or first letter navigation. Authors should follow known patterns.
+    Check out the [Resources](#resources) section for more info.
+
+For pure keyboard access issues, it's useful to also try without a screen reader, or with virtual
+browser mode off. Screen readers are not generally necessary to exhibit a keyboard bug, and keyboard
+access is actually implemented with HTML, not ARIA. After all, ARIA doesn't affect basic stuff like
+the keyboard or mouse behavior, it only lies to the screen reader about what's in the web page,
+what's currently focused, etc. 
+
+Keyboard bugs are almost always a bug in the web content, specifically in their HTML and JavaScript,
+not in ARIA.
+
+### ARIA bugs… why are there so many?
+
+There are many, many places where authors can get ARIA wrong, and each will lead to either complete
+breakage or subtle differences. The subtle ones are probably worse, because the author won't catch
+most of them on their own.
+
+After all, unless the author is an experienced screen reader user, something is going to go wrong in
+the ARIA. In our menubar, the author could think the "option" role was to be used when "menuitem"
+was correct. They could forget to use `aria-expanded`, forget to set and clear
+`aria-activedescendant` at the right times, or forget to have a menubar containing the other menus.
+And what about menu item counts? Usually menu items are presented by screen readers with something
+like "item 3 of 5" so that the user knows where they are. This is generally counted automatically by
+the browser, but in some cases, and in some browser - screen reader combinations, the wrong numbers
```

Comment:
```suggestion
like "item 3 of 5" so that the user knows where they are. The browser generally counts these automatically, but in some cases. In some browser-screen reader combinations, the wrong numbers
```